File: variables/currency.py
# Exchange rates to USD
colombian_peso = 0.00025
peruvian_sol = 0.27
brazilian_real = 0.21


# Ask user for left over non-usd currency
users_peso = float(input('How many pesos do you have left? '))
users_sol = float(input('How many sols do you have left? '))
users_real = float(input('How many reals do you have left? '))

# Calculate left over non-usd to usd
calc_peso = users_peso * colombian_peso
calc_sol = users_sol * peruvian_sol
calc_real = users_real * brazilian_real

# Calculate total in usd
total = calc_peso + calc_sol + calc_real

# Print total
print(total)

File: courses/javascript/02_variables/currency.js
// Currencies
let chineseYuan = 560
let japaneseYen = 2455
let southKoreanWon = 3280

// Exchange rates to USD
let yuanExchangeRate = 0.14;
let yenExchangeRate = 0.0067;
let wonExchangeRate = 0.00074;

// Calculate left over foreign currencies to USD
let calcYuan = chineseYuan * yuanExchangeRate;
let calcYen = japaneseYen * yenExchangeRate;
let calcWon = southKoreanWon * wonExchangeRate;

// Calculate total currency in USD
let totalCurrency = calcYuan + calcYen + calcWon;

// Print total USD currency
console.log(totalCurrency)